report/clients/standalone.js

```diff
@@ -26,6 +28,12 @@ function __initLighthouseReport__() {
       return document.documentElement.outerHTML;
     },
   });
+
+  // Fireworks look better in dark mode.
+  if (reportRootEl.querySelector('.lh-score100')) {
+    toggleDarkTheme(new DOM(document, reportRootEl), true);
```

Comment:
I like the idea of abstracting `DOM` behind `renderReport`. IMO it would be better to have a `enableDarkModeWithFireworks` option and moving this logic back to `ReportUIFeatures`. We could also use it for the report viewer.

There is an argument to remove this feature, it can be unexpected for some users https://github.com/GoogleChrome/lighthouse/issues/13631.

Reply:
It was unexpected because there were no fireworks. With fireworks, there is context, and it is clearly an easter egg.

> I like the idea of abstracting DOM behind renderReport.

If only the standalone report needs this, I'm fine keeping this internal.

Reply:
> If only the standalone report needs this, I'm fine keeping this internal.

What about report viewer?

Reply:
hmmm forgot about that... ok how's the new changeset?